app/main/views/services.py

```diff
@@ -452,17 +457,29 @@ def update_section_submission(framework_slug, lot_slug, service_id, section_id,
             "services/edit_submission_section.html",
             framework=framework,
             section=section,
+            next_section_name=get_next_section_name(content, section.id),
             service_data=update_data,
             service_id=service_id,
             one_service_limit=has_one_service_limit(lot_slug, framework['lots']),
+            return_to_summary=bool(request.args.get('return_to_summary')),
             errors=errors
         )
 
-    return redirect(url_for(".view_service_submission",
-                            framework_slug=framework['slug'],
-                            lot_slug=draft['lot'],
-                            service_id=service_id,
-                            _anchor=section_id))
+    return_to_summary = bool(request.args.get('return_to_summary'))
+    next_section = content.get_next_editable_section_id(section_id)
+
+    if next_section and not return_to_summary and request.form.get('continue_to_next_section'):
+        return redirect(url_for(".edit_service_submission",
```

Comment:
Its a question of how people add this information in a single flow vs many sittings. 

We've tested pretty heavily  (DOS applications and buyer journey) with the pattern of returning to the overview each time.

If people are using data from old services, then they're possibly going to go through multiple questions.

Other than that we would probably expect people to want to make sure they’re answering things correctly.

As a principle I favour clarity and making sure people do the right thing over worrying about people getting bored to clicking too much.

Plus we have the copy service feature which would make more sense to want to target certain questions rather than running through the complete service.

If the button takes you to the next question then we need the secondary button to return you to the overview, which I don't think is a great solution. The one contextual-research session I sat in on, the user didn't use the secondary 'Save and return to service overview'. Using the nice big green button to save any work they've done and then the breadcrumb to get to the overview. 

@TheDoubleK I don't think your experience reflects a typical user‘s, getting through the questions as quickly as you can isn't something users want to do. 

I also doubt many users will remember g-cloud 7 well enough and we’re making a change from DOS which is more recent. And means we’re probably going to have to change this behaviour once again?